pytest_dotenv/plugin.py

```diff
@@ -25,10 +27,18 @@ def pytest_addoption(parser):
 def pytest_load_initial_conftests(args, early_config, parser):
     _override = early_config.getini("env_override_existing_values")
     for filename in early_config.getini("env_files"):
+        _check_exists(filename)
         load_dotenv(find_dotenv(filename, usecwd=True), override=_override)
 
 
 def pytest_sessionstart(session):
     config = session.config
-    if config.getoption("envfile", default=None) is not None:
-        load_dotenv(dotenv_path=config.getoption("envfile"), override=True)
+    env_file = config.getoption("envfile", default=None)
+    if env_file is not None:
+        _check_exists(env_file)
+        load_dotenv(dotenv_path=env_file, override=True)
+
+
+def _check_exists(filename):
```

Comment:
Shall the plugin just print warnings to the user or completely stop the execution if something is not configured correctly?

Reply:
Also do you know who other py.test plugins behave in such a scenario?

Reply:
I do not know of any other plugins that have a similar scenario. I'll try and look some up.

The way I implemented it continues execution but prints to stderr (which should highlight for certain terminals). `load_dotenv` does nothing if it cannot find the file. So it is currently silently failing. The warning should make it more obvious.